style.css

```diff
@@ -254,3 +256,172 @@ textarea:hover {
 input:hover {
   border: 1px solid #253858;
 }
+
+@media screen and (min-width: 768px) {
+  .main {
+    background: url(images/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-color: #fff;
+    border-bottom-left-radius: 100px;
+  }
+
+  body {
+    background-color: #e5e5e5;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-between;
+    padding: 40px 190px 40px 90px;
+    background-color: #fff;
+  }
+
+  .nav-links {
+    display: flex;
+  }
+
+  .nav-links ul {
+    display: flex;
+    list-style: none;
+    gap: 35px;
+  }
+
+  .nav-links ul li a {
+    color: #344563;
+    text-decoration: none;
+    font-size: 15px;
+  }
+
+  .logo {
+    font-size: 18px;
+  }
+
+  h1 {
+    font-size: 45px;
+  }
+
+  .fa-bars {
+    visibility: hidden;
+  }
+
+  .presentation {
+    font-size: 20px;
+    padding-left: 0;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .lang {
+    padding-top: 0;
+    padding-bottom: 0;
+  }
+
+  .connect {
+    font-size: 16px;
+    padding-left: 0;
+  }
+
+  .icons ul li a {
+    font-size: 20px;
+  }
+
+  .icons {
+    margin-left: 0;
+    padding: 12px;
+  }
+
+  .content {
+    width: 85%;
+  }
+
+  .works {
+    background-color: #fff;
+    display: flex;
+    width: 70%;
+    margin-bottom: 100px;
+    margin-top: 100px;
+    height: 410px;
+  }
```

Comment:
- [ ] As requested by the previous CR, please 🙏🏽 fix on the styles in the Figma design, kindly correct the styles of the buttons 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/236584602-81d422ba-f701-4935-90af-125a44ceffe8.png) | ![image](https://user-images.githubusercontent.com/58642949/236584642-7564545a-8319-48bd-810a-6a398dfea99c.png) |